tests/e2e/agileTravel.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { test, expect } from '@playwright/test';
+import { AgileTravelPage } from '../pom/agileTravel.page';
+
+let travelPage: AgileTravelPage;
+
+test.beforeEach(async ({ page }) => {
+  travelPage = new AgileTravelPage(page);
+
+  await travelPage.gotoLoginPage();
+  await travelPage.login('agileway', 'testW1se');
+
+  await expect(page.getByRole('heading', { name: 'Select Flight' })).toBeVisible();
+  await travelPage.selectFlightPort('New York', 'Sydney');
+  await travelPage.selectDates('082025', '15', '092025', '05');
+});
+
+test('Successful login to Agile Travel site', async () => {
```

Comment:
I would reconsider if such test would benefit us runing it all the time as separate test. Loging-in is used/required in every test: without logging in user would not be able to do any further steps